<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Venta_Bus_Documentos</class>
 <widget class="QMainWindow" name="Venta_Bus_Documentos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>769</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Text_Buscador">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Buscar">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabla_Coincidencia">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>681</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
